pep-0674.rst

```diff
@@ -10,23 +10,24 @@ Python-Version: 3.11
 Abstract
 ========
 
-Incompatible C API change disallowing using macros as an l-value to:
+Incompatible C API change disallowing using macros, especially
+``Py_TYPE()`` and ``Py_SIZE()``, as l-values to:
 
-* Allow evolving CPython internals;
+* Allow evolving CPython internals (change the ``PyObject`` structure);
 * Ease the C API implementation on other Python implementation;
 * Help migrating existing C extensions to the HPy API.
 
-Only 9 out of the top 5000 PyPI projects (0.2%) are affected by 2 macro
-changes. An additional 22 projects just have to regenerate their Cython
+Only 7 out of the top 5000 PyPI projects (0.1%) are affected by this
+PEP. An additional 22 projects just have to regenerate their Cython
```

Comment:
The number is listed as 23 projects below

Reply:
Fixed in commit e1b48f92dae4c0364470b92ccea15f90e2db64d9.

Reply:
Thanks!